src/Sentry/Sentry.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(SolutionName)' != 'Sentry.Unity'">
-    <TargetFrameworks>net6.0;net5.0;netcoreapp3.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>net8.0;net6.0;net5.0;netcoreapp3.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
```

Comment:
Also, do we need to add net8.0 at all? Because I think this was a recent addition in this PR so if it's just for sake of adding it, I'd prefer not to because it makes builds even slower.

Reply:
I've removed this temporarily because of test failures in CI. If we need it (in this PR), feel free to bring it back.